### PR TITLE
add package.json

### PR DESCRIPTION
**Pull request by benpptung**
Any chance to put tap.js in npm ?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Dependency directory
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
+node_modules
+
+.idea
```
Comment (alexgibson) [inline .gitignore]: You should probably add this to your global .gitignore instead of to this repo.

**Comment by alexgibson** [comment]
Looks good, please just remove the webstorm stuff. Thanks!

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.idea
```
Comment (alexgibson) [inline .npmignore]: Same here.

**Comment by benpptung** [comment]
Oops! I didn't mean to pull request the .ignore files commit. I push the commit https://github.com/benpptung/tap.js/commit/326ebf5408485e57dbff167ec5af52cfa2e36c3e after I created the pull request. Apologized that I am not familiar to Github, will try to figure out how to remove it. or can you simply ignore it? :)

**Pull request closed without merging**